docs/fedora.md

```diff
@@ -156,7 +156,7 @@ Once that's done, we can build a sysext:
 ```bash
 sudo PYTHONPATH=$PWD/../mkosi python -m mkosi --default fedora.mkosi -f \
   --image-version=$KVER-ssh \
-  --base-image=mkosi.output/initrd_$KVER.d \
+  --base-image=mkosi.output/initrd_$KVER \
```

Comment:
This doesn't seem to be configured anywhere explicitly. I used `.d` locally, but I think I specified it explicitly. If `mkosi` uses a name without `.d` automatically, let's just do that.